### PR TITLE
Issue55plot

### PR DESCRIPTION
**Pull request by Kleyt0n**

### CODE REVIEW

File: holisticai/bias/plots/_bias_report_plots.py [hunk 1]
```diff
@@ -0,0 +1,543 @@
...
+            axes[row, col].set_xticklabels([])
+
+            if i == len(metric_names) - 1 and i % cols != cols - 1:
+                for j in range(i % cols + 1, cols):
+                    axes[row, j].axis("off")
+            elif i == len(metric_names) - 1 and i % cols == cols - 1:
+                for j in range(cols):
+                    axes[row + 1, j].axis("off")
+
+            if name == "RMSE Ratio":
```
Comment (crismunoz) [inline holisticai/bias/plots/_bias_report_plots.py]: store the range values in a dictionary (reduce this code)

File: holisticai/bias/plots/_bias_report_plots.py [hunk 2]
```diff
@@ -0,0 +1,543 @@
...
+                    ],
+                    bbox_to_anchor=(-0.85, 1),
+                    loc="upper left",
+                    fontsize=12,
+                )
+
+            plt.tight_layout()
+
+    else:
+        for i, name in enumerate(metric_names):
```
Comment (crismunoz) [inline holisticai/bias/plots/_bias_report_plots.py]: No duplicated code

File: holisticai/bias/mitigation/inprocessing/variational_fair_clustering/algorithm_utils/_method_utils.py [hunk 3]
```diff
@@ -147,10 +156,15 @@ def _init_centroids(
         n_samples = X.shape[0]
 
     if isinstance(init, str) and init == "k-means++":
+        if check_version(SKLEARN_CURRENT_VERSION) >= check_version("1.3.0"):
```
Comment (crismunoz) [inline holisticai/bias/mitigation/inprocessing/variational_fair_clustering/algorithm_utils/_method_utils.py:160]:
```python
  kargs = {}
  if check_version(SKLEARN_CURRENT_VERSION) >= check_version("1.3.0"):
            kargs = {"sample_weight": np.ones(shape=(X.shape[0],))}

```

**Review by crismunoz — CHANGES_REQUESTED** [review]
good work, please check the commentaries

File: holisticai/bias/mitigation/inprocessing/variational_fair_clustering/algorithm_utils/_method_utils.py (hunk 3 above)
Comment (kleyt0n) [reply holisticai/bias/mitigation/inprocessing/variational_fair_clustering/algorithm_utils/_method_utils.py:160]: Done

File: holisticai/bias/plots/_bias_report_plots.py (hunk 2 above)
Comment (kleyt0n) [reply holisticai/bias/plots/_bias_report_plots.py]: Done.

**Review by crismunoz — CHANGES_REQUESTED** [review]
Too much duplicated code!

File: holisticai/bias/plots/_bias_report_plots.py (hunk 1 above)
Comment (kleyt0n) [reply holisticai/bias/plots/_bias_report_plots.py]: Done.

**Review by crismunoz — APPROVED** [review]
nice!

**Pull request merged**